src/Tools/XmlDocumentationsUtility/Test/ZeroTouchModuleTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NodeDocumentationUtility;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace XmlDocumentationsUtility.Test
+{
+    [TestFixture]
+    class ZeroTouchModuleTests
+    {
+        private ZeroTouchModule module = null;
+
+        [SetUp]
+        public void SetUp()
+        {
+            module = new ZeroTouchModule("ProtoGeometry.dll");
+        }
+
+        [Test]
+        public void TypeExists()
+        {
+            Assert.IsFalse(module.TypeExists("Analysis.IAnalysisData"));
```

Comment:
Use some specific class which is present in ProtoGeometry but that is not available in ZeroTouchModule, such as HostFactory or any other class with [SuppressImportInVM] attribute.